backend/api/Controllers/MissionController.cs

```diff
@@ -91,9 +92,13 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string id)
     {
         try
         {
-            string filePath = await _mapService.FetchMapImage(id);
-            var returnFile = PhysicalFile(filePath, "image/png");
-            return returnFile;
+            Image mapImage = await _mapService.FetchMapImage(id);
+
+            using (var ms = new MemoryStream())
+            {
+                mapImage.Save(ms, mapImage.RawFormat);
+                return File(ms.ToArray(), "image/png");
```

Comment:
Avoid abbreviations in var names:

```suggestion
            using (var memoryStream = new MemoryStream())
            {
                mapImage.Save(memoryStream, mapImage.RawFormat);
                return File(memoryStream.ToArray(), "image/png");
```